frontend/src/metabase/public/components/widgets/EmbedModalContent.unit.spec.tsx

```diff
@@ -78,8 +78,14 @@ describe("EmbedModalContent", () => {
     );
 
     openEmbedModal();
-    expect(screen.getByText("My param")).toBeInTheDocument();
-    expect(screen.getByLabelText("My param")).toHaveTextContent("Locked");
+    const parametersSection = screen
+      .getByRole("heading", { name: "Parameters" })
+      // eslint-disable-next-line testing-library/no-node-access
```

Comment:
This feels a bit awkward but I could find we disable this ESLint rule is some other places as well. So I think it might be fine 🤷

Reply:
If you need to differentiate between these sections you can use `role="section"` with `aria-label` for the section(s)

Reply:
Yeah, this is the problem I talked about in our team sync that makes me want to create an RFC for section accessibility.  I think your solution might work.